### ISSUE TITLE
How to reuse tags from other catalogue

### ISSUE THREAD

**Issue by TobiasNx** [pkiraly/qa-catalogue#507]
Hi,
we try to reuse the DNB Tags in out hbz catalogue.

I tried it like this, but it does not work: https://github.com/hbz/qa-catalogue/commit/517869aecd95913a937935427edb4c305c4bef24#diff-6ab233441d2457fd4d4370036d454637fc9ac8133b4d0226ffffa3b206391d18

Is there a wys to reuse the dnb Tags?

Thanks

**Comment by pkiraly** [comment]
@TobiasNx It is not possible now, unless you copy the classes. But this issue might have a solution: to accept multiple `marcVersion`s, in your case it would be something like: `--marcVersion DNB,HBZ` - with the functionality that the later overwrites the previous, so let's suppose DNB defined 900, 901 and 902, HBZ defines 902 and 903. The result will be 900, 901 from DNB and 902 and 903 from HBZ. DNB's 902 is overwriten by HBZ's 902.
Would it be acceptable for you?

**Comment by TobiasNx** [comment]
Thanks for getting back. Yes i think this could solve the issue. We will try it: https://github.com/hbz/qa-catalogue/commit/05906fbc8fd3b1446970e1090454840113d3f3bc

**Comment by nichtich** [comment]
#368 would help to solve this as well. Having to create and reuse Java classes is more complex than putting together an Avram JSON file.

**Comment by TobiasNx** [comment]
> #368 would help to solve this as well. Having to create and reuse Java classes is more complex than putting together an Avram JSON file.

Yes, I think so too and I still have a lot of Java Classes to write: https://github.com/hbz/qa-catalogue/pull/1#issuecomment-2314668762

**Comment by Phu2** [comment]
I tried using `--marcVersion DNB,HBZ` in our catalogue, see https://github.com/hbz/qa-catalogue/blob/createAdditionalFieldsForHbzPublishingCatalogue/catalogues/hbz-update.sh#L14 but got 

```
Aug 28, 2024 10:11:04 AM de.gwdg.metadataqa.marc.cli.ValidatorCli main
SEVERE: ERROR. Unrecognized marcVersion parameter value: 'DNB,HBZ'
```

Am i missing something here?

**Comment by pkiraly** [comment]
Dear @Phu2, here we just collected ideas, it is not yet implemented. I am working on it, and I will ping you when it will be ready (surely later this week).

**Comment by Phu2** [comment]
Ah i got that wrong, sorry. No hurry.
